Clients/GUI/Pages/UserInfo.razor

```diff
@@ -0,0 +1,133 @@
+    <EditForm Model="@user">
+        <DataAnnotationsValidator />
+        <div class="form-group">
+            <label for="Name">Name:</label>
+            <InputText class="input-group-text" disabled="@IsEditModeDisabled" @bind-Value="user.Name" />
+        </div>
+        <div class="form-group">
+            <label for="Surname">Surname:</label>
+            <InputText class="input-group-text" disabled="@IsEditModeDisabled" @bind-Value="user.Surname" />
+        </div>
+        <div class="form-group">
+            <label for="Birthday">Birthday:</label>
+            <InputDate class="input-group-text" disabled="@IsEditModeDisabled" @bind-Value="@user.DateOfBirth" />
+        </div>
+        <div class="form-group">
+            <label for="Email">Email:</label>
+            <InputText class="input-group-text" disabled="disabled" @bind-Value="@user.Email" />
+        </div>
+        <div class="form-group" style="@(IsPasswordBeingChanged ? "" : "height: 0; color: transparent")">
+            <label for="OldPassword">Enter old password: </label>
+            <InputText type="@(IsPasswordBeingChanged ? "password" : "hidden")" class="input-group-text" disabled="@(!IsPasswordBeingChanged)" @bind-Value="@user.OldPassword" />
+        </div>
+        <div class="form-group" style="@(IsPasswordBeingChanged ? "" : "height: 0; color: transparent")">
+            <label for="NewPassword">Enter new password: </label>
+            <InputText type="@(IsPasswordBeingChanged ? "password" : "hidden")" class="input-group-text" disabled="@(!IsPasswordBeingChanged)" @bind-Value="@user.NewPassword" />
+        </div>
+        <br />
+        <input type="@(IsEditModeDisabled ? "hidden" : "button")" @onclick="@ChangePasswordClickHandler" class="form-control col-3 button btn-primary" value="@(IsPasswordBeingChanged ? "Cancel changing password" : "Change password")" />
+        <br />
+        <ValidationSummary />
+        <h4 style="display:@(ErrorOccured ? "block" : "hidden"); color:red">@ErrorMessage</h4>
+        <input type="button" @onclick="@EditInfoClickHandler" class="form-control col-3 button btn-primary" value="@(IsEditModeDisabled ? "Edit" : "Save changes")" />
+    </EditForm>
+</div>
+
+@code {
+    public bool IsEditModeDisabled { get; set; }
+    public bool IsPasswordBeingChanged { get; set; }
+    public bool PopupVisible { get; set; }
+    public string ErrorMessage { get; set; }
+    public bool ErrorOccured { get; set; }
+    private UserInfoViewModel user;
+    private Guid currentId;
+
+    private void EditInfoClickHandler()
+    {
+        if (IsEditModeDisabled)
+        {
+            IsEditModeDisabled = false;
+        }
+        else // Trying to save changes
+        {
+            // Edit is inprocess in other team
+            try
+            {
+                var userInfo = new DTO.UserInformation(user.Name, user.Surname, user.DateOfBirth);
+
+                // This is in process (user/edit endpoint is another team's task)
```

Comment:
Better to change with TODO, I think

Reply:
Technically it's already a current task for another team

Reply:
Ok